medicinal_plants/lib/src/screens/landing-screen.dart

```diff
@@ -39,18 +41,66 @@ class LandingScreen extends StatelessWidget {
             ),
           ),
 
-          Container(
-            margin: EdgeInsets.zero,
-            padding: EdgeInsets.zero,
-            width: width,
-            height: height,
+          Center(
             child: Text(
               'Medicinal Plants',
-              style: TextStyle(
-                
+              style: GoogleFonts.poppins(
+                color: Colors.white,
+                fontWeight: FontWeight.w500,
+                fontSize: 40,
+              )
+            ),
+          ),
+
+          Center(
+            child: Container(
+              margin: EdgeInsets.only(top: height*.3),
+              height: height*.10,
+              width: width*.70,
+              child: Center(
+                child: Text(
+                  'Find medicinal plants using our app',
+                  maxLines: 2,
+                  textAlign: TextAlign.center,
+                  style: GoogleFonts.poppins(
+                    color: Colors.white,
+                    fontWeight: FontWeight.w500,
+                    fontSize: 20,
+                  )
+                ),
+              ),
+            ),
+          ),
+
+          Center(
+            child: GestureDetector(
+              onTap: () {
+              Navigator.pushNamed(context, '/home');
+            },
+              child: Container(
+                margin: EdgeInsets.only(top: height*.8),
+                height: height*.07,
+                width: width*.70,
+                decoration: BoxDecoration(
+                  borderRadius: BorderRadius.circular(10),
+                  color: buttonColor,
+                ),
+                child: Center(
+                  child: Text(
+                    'Get Started',
```

Comment:
create a folder under constants folder and name it "properties"
create a file inside it and name it: medicinal_properties.dart

```
class MedicinalProperties {

    static const GET_STARTED = "Get Started"

}

```

then you can use it in this line like:  MedicinalProperties.GET_STARTED